corehq/apps/reports/views.py

```diff
@@ -786,10 +799,12 @@ def scheduled_report_form(self):
         form.fields['config_ids'].choices = self.config_choices
         form.fields['recipient_emails'].choices = [(e, e) for e in web_user_emails]
 
-        form.fields['hour'].help_text = "This scheduled report's timezone is %s (%s GMT)" % \
+        form.fields['hour'].help_text = _("This scheduled report's timezone is %s (%s GMT)") % \
```

Comment:
Super ultra nit (and I appreciate that you didn't write this): Since 1963, UTC is [a standard](https://en.wikipedia.org/wiki/Coordinated_Universal_Time#Uses) and GMT is [a timezone](https://en.wikipedia.org/wiki/Greenwich_Mean_Time#Time_zone). e.g. For a project in, say, Iceland, you'd say "This scheduled report's timezone is GMT (UTC+00:00)"
```suggestion
        form.fields['hour'].help_text = _("This scheduled report's timezone is %s (UTC%s)") % \
```